docs/MISSION_AND_VALUES.md

```diff
@@ -0,0 +1,57 @@
+
+- being **well documented** with **comprehensive tutorials and examples**. All functions in our API have thorough docstrings clarifying expected inputs and outputs, and we maintain a separate [tutorials and example website](http://napari.org) to explain different use cases and working modes.
+
+- providing **GUI access** to all critical functionality so napari can be used by people with no coding experience.
+
+- being **interactive** and **highly performant** in order to support very large data sets.
+
+- providing a **consistent and stable API** to enable plugin developers to build on top of napari without their code constantly breaking and to enable advanced users to build out sophisticated Python workflows.
+
+- **ensuring correctness**. We strive for complete test coverage of both the code and GUI, with all code reviewed by a core developer before being included in the repository.
+
+
+## Our values
+
+- We are **inclusive**. We welcome and mentor newcomers who are making their first contribution and strive to grow our most dedicated contributors into [core developers](CORE_DEV_GUIDE.md).
+
+- We are **community-driven**. We respond to feature requests and proposals on our [issue tracker](https://github.com/napari/napari/issues), making decisions that are driven by our users’ requirements, not by the whims of the core team.
+
+- We provide **a tool others can build on** and making it easy for people to develop and share plugins that extend napari's functionality and to develop fully custom applications that consume napari.
+
+- We serve **scientific applications** primarily, over “consumer” image editing in the vein of Photoshop or GIMP. This often means prioritizing n-dimensional data support, and rejecting implementations of “flashy” features that have little scientific value.
+
+- We are **domain agnostic** within the sciences. Functionality that is highly specific to particular scientific domains belongs in plugins, whereas functionality that cuts across many domains and is likely to be widely used belongs inside napari.
+
+- We **focus on visualization**, leaving image analysis functionality and highly custom file IO to plugins.
+
+- We value **simple, readable implementations**. Readable code that is easy to understand, for newcomers and maintainers alike, makes it easier to contribute new code as well as prevent bugs.
+
+- We value **education and documentation**. All functions should have docstrings, preferably with examples, and major functionality should be explained in our [tutorials](https://github.com/napari/napari-tutorials). Core developers can take an active role in finishing documentation examples.
+
+- We **don’t do magic**. We support NumPy array like objects and we prefer to educate users rather than make decisions on their behalf. This does not preclude the use of sensible defaults.
```

Comment:
This made sense for scikit-image, but not so much for napari, where we are doing magic left, right, and centre. (And you'll notice we've paid a price for it, which reinforces my belief in this principle to be honest! =) Perhaps "we minimize magic, and always provide a way for users to opt out of all magical behaviour". The opt-out principle would have prevented some of our pyramid issues. What do you think?

Reply:
i think it's actually still a good value and something that can help guide us in our decision making, even if we havn't lived it 100% so far - maybe all the more reason to keep it in here, but make it a little more napari centric

Reply:
i've updated to capture this